test/integration/targets/interpreter_discovery_python/tasks/main.yml

```diff
@@ -161,7 +161,7 @@
       # rhel 8
       - (auto_out.ansible_facts.discovered_interpreter_python == '/usr/libexec/platform-python' and distro_major_version is version('8','==')) or distro_major_version is version('8','!=')
```

Comment:
We should probably nuke the pre-RHEL9 and old Ubuntu tests here since they're not getting run in CI, and they'd fail anyway since the distro map has been neutered...

Reply:
done